stable/gcpdisk-csi-driver/templates/csi-gcpdisk-controller.yaml

```diff
@@ -15,52 +15,48 @@ spec:
 {{ include "gcpdisk.labels" . | indent 6 }}
         app: gcp-compute-persistent-disk-csi-driver
     spec:
-      # Host network must be used for interaction with Workload Identity in GKE
-      # since it replaces GCE Metadata Server with GKE Metadata Server. Remove
-      # this requirement when issue is resolved and before any exposure of
-      # metrics ports
-      hostNetwork: true
       serviceAccountName: csi-gce-pd-controller-sa
       nodeSelector:
         beta.kubernetes.io/os: linux
       priorityClassName: system-cluster-critical
+{{- if .Values.tolerations }}
+      tolerations:
+{{ toYaml .Values.tolerations | indent 8 }}
+{{- end }}
       containers:
         - name: csi-provisioner
           image: {{ .Values.provisioner.image.repository }}:{{ .Values.provisioner.image.tag }}
           args:
+            - --v=5
```

Comment:
Configurable via `values.yaml`?

Reply:
Just noticed it seems to be the same setting in the other csi drivers. Maybe it should be a setting available for all of them? wdyt?

Reply:
could be done for all sure, but it is not right now.
We can tackle it in another ticket I will create.

Reply:
https://jira.d2iq.com/browse/D2IQ-68803